App/SB/views/PhotoDetail/index.js

```diff
@@ -194,19 +218,21 @@ const mapStateToProps = (state, ownProps) => {
   }).sort((a, b) => b - a)
 
   const path = thread.name === 'default' ? '/photo' : '/thumb'
+  const source = photo ? {url: 'file://' + photo.id + '.png'} : {url: 'file://.png'}
+
   return {
     photo,
-    date: photo.date.split('T')[0],
-    key: photo.id + path,
-    source: {url: 'file://' + photo.id + '.png'}, // <-- in case RN uses to know things
+    date: photo && photo.date ? photo.date.split('T')[0] : '',
+    key: photo && photo.id ? photo.id + path : path,
```

Comment:
all this is just making it safe the if the view doesn't unmount in time before the image remove request is done that it wont error out